x-pack/plugins/enterprise_search/public/applications/app_search/components/search_ui/components/search_ui_form.tsx

```diff
@@ -119,8 +127,8 @@ export const SearchUIForm: React.FC = () => {
         />
       </EuiFormRow>
       <EuiButton
```

Comment:
Rather than being a link as before, this is not a form with a submit button so that we can POST to the new route.